parachain/pallets/inbound-queue/Cargo.toml

```diff
@@ -26,17 +25,17 @@ sp-io = { git = "https://github.com/paritytech/substrate.git", branch = "master"
 sp-runtime = { git = "https://github.com/paritytech/substrate.git", branch = "master", default-features = false }
 
 xcm = { git = "https://github.com/paritytech/polkadot.git", branch = "master", default-features = false }
+xcm-builder = { git = "https://github.com/paritytech/polkadot.git", branch = "master", default-features = false }
 bp-runtime = { git = "https://github.com/Snowfork/cumulus.git", branch = "snowbridge", default-features = false }
 
 snowbridge-core = { path = "../../primitives/core", default-features = false }
 snowbridge-ethereum = { path = "../../primitives/ethereum", default-features = false }
 snowbridge-router-primitives = { path = "../../primitives/router", default-features = false }
 ethabi = { git = "https://github.com/Snowfork/ethabi-decode.git", package = "ethabi-decode", branch = "master", default-features = false }
 
-snowbridge-beacon-primitives = { path = "../../primitives/beacon", optional = true }
+snowbridge-beacon-primitives = { path = "../../primitives/beacon", default-features = false, optional = true }
```

Comment:
This was the line that fixed it for me.

Reply:
Exactly! That's key change.

Reply:
This was my bad, I added this when adding benchmarks for the inbound queue. It worked back then, but I can see how it could cause issues.

Reply:
@claravanstaden also, this dependency is marked as `optional` yet there is no feature that requires it, which seems wrong.

Reply:
`runtime-benchmarks` requires the feature, and it is also required for tests.

Reply:
Oops, yes I see now. `runtime-benchmarks` requires `snowbridge-beacon-primitives/runtime-benchmarks`, which is provided by that dependency :+1: